nixos/modules/services/networking/expressvpn.nix

```diff
@@ -0,0 +1,20 @@
+{ config, lib, pkgs, ... }:
+
+{
+  options.services.expressvpn.enable = lib.mkOption {
+    type = lib.types.bool;
+    default = false;
+    description = ''
+      Enable the ExpressVPN daemon.
+    '';
+  };
+
+  config = lib.mkIf config.services.expressvpn.enable {
+    boot.kernelModules = [ "tun" ];
+
+    environment.systemPackages = [ pkgs.expressvpn ];
+    systemd.packages = [ pkgs.expressvpn ];
```

Comment:
I think this also needs `systemd.services.expressvpn.wantedBy = [ "multi-user.target" ]` to make it work on `nixos-rebuild switch` or am I mistaken here?

Reply:
It's implemented in the systemd file that's shipped with the binary itself, so I haven't explicitly added it.